### PR TITLE
Add translations

### PR DESCRIPTION
**Pull request by LISBON11**

### CODE REVIEW

File: common.blocks/add-form/_type/add-form_type_post.bemtree.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
-block('add-form').mod('type', 'post').elem('fields').content()(node => {
-    const block = node.block;
+block('add-form').mod('type', 'post').elem('fields').content()(function() {
```
Comment (tadatuta) [inline common.blocks/add-form/_type/add-form_type_post.bemtree.js]: any reason to replace arrow function?

**Comment by LISBON11** [comment]
issue__toggle-resolved -  'Переключить между решено и не решено'

File: common.blocks/add-form/add-form.bemtree.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 block('add-form')(
     content()(function() {
-        const block = this.block;
+        const block = this.block,
+            i18n = this.require('i18n');
```
Comment (tadatuta) [inline common.blocks/add-form/add-form.bemtree.js]: let's use `const` on each line

File: common.blocks/add-form/_type/add-form_type_post.bemtree.js (hunk 1 above)
Comment (volkov97) [reply common.blocks/add-form/_type/add-form_type_post.bemtree.js]: node.i18n is undefined

Comment (tadatuta) [reply common.blocks/add-form/_type/add-form_type_post.bemtree.js]: `node.require('i18n')`?

Comment (volkov97) [reply common.blocks/add-form/_type/add-form_type_post.bemtree.js]: also not working

Comment (skad0) [reply common.blocks/add-form/_type/add-form_type_post.bemtree.js]: update branch and `i18n` will be available in `node`

Comment (tadatuta) [reply common.blocks/add-form/_type/add-form_type_post.bemtree.js]: don't forget to update npm deps (remember about lock file as well)

**Review by tadatuta — APPROVED** [review]

**Pull request merged**